lib/linguist/languages.yml

```
@@ -7622,6 +7622,18 @@ jq:
   - ".jq"
   tm_scope: source.jq
   language_id: 905371884
+just:
+  type: programming
+  aliases:
+  - Justfile
+  color: "#384d54"
+  tm_scope: source.justfile
```

This scope is wrong.

```suggestion
  tm_scope: source.just
```